modules/ROOT/pages/tutorial-updates.adoc

```diff
@@ -103,67 +106,131 @@ virt-install --name=fcos --vcpus=2 --ram=2048 --import \
+● zincati.service - Zincati Update Agent
+     Loaded: loaded (/usr/lib/systemd/system/zincati.service; enabled; vendor preset: enabled)
+     Active: active (running) since Fri 2020-08-07 17:42:14 UTC; 51s ago
+       Docs: https://github.com/coreos/zincati
+   Main PID: 889 (zincati)
+      Tasks: 5 (limit: 2290)
+     Memory: 17.8M
+     CGroup: /system.slice/zincati.service
+             ├─ 889 /usr/libexec/zincati agent -v
+             └─4463 rpm-ostree finalize-deployment a3b08ee51b1d950afd9d0d73f32d5424ad52c7703a6b5830e0dc11c3a682d869
+
+Aug 07 17:42:14 localhost systemd[1]: Started Zincati Update Agent.
+Aug 07 17:42:14 localhost zincati[889]: [INFO ] starting update agent (zincati 0.0.11)
+Aug 07 17:42:17 localhost zincati[889]: [INFO ] Cincinnati service: https://updates.coreos.fedoraproject.org
+Aug 07 17:42:17 localhost zincati[889]: [INFO ] agent running on node '2ce419a50b0a4a4f8b10e5e9cf487b68', in update group 'default'
+Aug 07 17:42:17 localhost zincati[889]: [INFO ] initialization complete, auto-updates logic enabled
+Aug 07 17:42:18 localhost zincati[889]: [INFO ] target release '32.20200715.3.0' selected, proceeding to stage it
+Aug 07 17:43:05 localhost zincati[889]: [INFO ] staged deployment '32.20200715.3.0' available, proceeding to finalize it
+----
+
+Shortly after, the system should reboot to apply the latest update.
+
+When we log back in we can view the current version of Fedora CoreOS is now `32.20200715.3.0`. The rpm-ostree status output will also show the older version, which still exists in case we need to rollback:
 
 [source,bash]
 ----
-$ rpm-ostree status
+[core@localhost ~]$ rpm-ostree status
 State: idle
-AutomaticUpdates: disabled
 Deployments:
 ● ostree://fedora:fedora/x86_64/coreos/stable
-                   Version: 31.20200113.3.1 (2020-01-14T00:20:15Z)
-                    Commit: f480038412cba26ab010d2cd5a09ddec736204a6e9faa8370edaa943cf33c932
-              GPGSignature: Valid signature by 7D22D5867F2A4236474BF7B850CB390B3C3359C4
+                   Version: 32.20200715.3.0 (2020-07-27T11:36:29Z)
+                    Commit: a3b08ee51b1d950afd9d0d73f32d5424ad52c7703a6b5830e0dc11c3a682d869
+              GPGSignature: Valid signature by 97A1AE57C3A2372CCA3A4ABA6C13026D12C944D0
 
   ostree://fedora:fedora/x86_64/coreos/stable
-                   Version: 31.20200108.3.0 (2020-01-09T21:51:07Z)
-                    Commit: 113aa27efe1bbcf6324af7423f64ef7deb0acbf21b928faec84bf66a60a5c933
-              GPGSignature: Valid signature by 7D22D5867F2A4236474BF7B850CB390B3C3359C4
+                   Version: 32.20200629.3.0 (2020-07-10T17:58:03Z)
+                    Commit: 6df95bdb2fe2d36e091d4d18e3844fa84ce4b80ea3bd0947db5d7a286ff41890
+              GPGSignature: Valid signature by 97A1AE57C3A2372CCA3A4ABA6C13026D12C944D0
 ----
 
-*NOTE:* The currently booted deployment is denoted by the `●` character.
+[NOTE]
+====
+The currently booted deployment is denoted by the `●` character.
+====
```

Comment:
```suggestion
NOTE: The currently booted deployment is denoted by the `●` character.
```

I think Asciidoc will treat what you had and what I suggested equivalently. Do you agree? If so, which one do you prefer?

Reply:
How indeed. I was trying to make that Uniform across the docs sources. Will update.